collector/collector.go

```diff
@@ -34,6 +35,13 @@ func getValidLabelName(str string) string {
 	return prometheusMetricNameInvalidCharactersRegex.ReplaceAllString(strings.Replace(str, "-", "_", -1), "")
 }
 
+func b2f(cond bool) float64 {
```

Comment:
Add a comment before the function definition to explain its purpose (i.e. convert bool to float value)